ajax/itilfollowup.php

```diff
@@ -86,5 +86,33 @@
 // Render template content using twig
 $template->fields['content'] = $template->getRenderedContent($parent);
 
+//load requesttypes name (use to create OPTION dom)
+//need when template is used and when GLPI preselected type if defined
+$template->fields['requesttypes_name'] = "";
+if ($template->fields['requesttypes_id']) {
+    $dbUtils = new DbUtils();
+    $entityRestrict = $dbUtils->getEntitiesRestrictCriteria(getTableForItemType(RequestType::getType()), "", $parent->fields['entities_id'], true, false);
+    if (count($entityRestrict)) {
+        $entityRestrict = [$entityRestrict];
+    }
```

Comment:
I do not know why `DbUtils` is not doing that, but `getEntitiesRestrictCriteria` is already encapsulating the result, so code could be simplier.

```suggestion
    $entityRestrict = getEntitiesRestrictCriteria(getTableForItemType(RequestType::getType()), "", $parent->fields['entities_id'], true, false);
```